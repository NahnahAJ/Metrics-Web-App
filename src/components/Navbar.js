import React from 'react';
import { useNavigate } from 'react-router-dom';
import { FaAngleLeft, FaMicrophone } from 'react-icons/fa';
import { AiOutlineSetting } from 'react-icons/ai';
import '../styles/Navbar.modules.css';
// FaSistrix search

const Navbar = () => {
  const navigate = useNavigate();
  return (
    <nav className="consDisplay displays">
      <button className="backBtn iconSize" type="button" key={Date.now()} onClick={() => navigate(-1)} aria-label="Back"><FaAngleLeft /></button>
      <h2>Vintage Collections</h2>
      <div className="iconsCtn displays">
        <span className="iconSize"><FaMicrophone /></span>
        <span className="iconSize"><AiOutlineSetting /></span>
      </div>
    </nav>
  );
};

export default Navbar;
