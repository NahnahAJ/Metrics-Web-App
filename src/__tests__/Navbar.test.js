import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configureStore';
import Navbar from '../components/Navbar';

test('renders the Navbar component snapshot', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Provider store={store}>
        <Navbar />
      </Provider>
      ,
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
