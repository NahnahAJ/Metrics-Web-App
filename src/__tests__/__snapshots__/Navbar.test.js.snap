// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Navbar component snapshot 1`] = `
<DocumentFragment>
  <div
    class="consDisplay"
  >
    <button
      type="button"
    >
      back key
    </button>
    <h2>
      Components
    </h2>
    <p>
      mic
    </p>
    <p>
      settings
    </p>
  </div>
  ,
</DocumentFragment>
`;
